objState.rdf

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="owl2html.xsl"?>
+
+<!DOCTYPE rdf:RDF [
+    <!ENTITY owl "http://www.w3.org/2002/07/owl#" >
+    <!ENTITY xsd "http://www.w3.org/2001/XMLSchema#" >
+    <!ENTITY rdfs "http://www.w3.org/2000/01/rdf-schema#" >
+    <!ENTITY rdf "http://www.w3.org/1999/02/22-rdf-syntax-ns#" >
+]>
+
+
+<rdf:RDF xmlns="http://fedora.info/definitions/1/0/access/objState"
+     xml:base="http://fedora.info/definitions/1/0/access"
+     xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+     xmlns:owl="http://www.w3.org/2002/07/owl#"
+     xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
+     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
+
+     <owl:Ontology rdf:about="http://fedora.info/definitions/1/0/access/objState">
+        <rdfs:label xml:lang="en">Fedora Commons Repository - Object Status Property</rdfs:label>
+        <rdfs:comment xml:lang="en">Object status property</rdfs:comment>
+        <owl:versionInfo>v4/2015/03/10</owl:versionInfo>
+    </owl:Ontology>
+
+    <owl:ObjectProperty rdf:about="http://fedora.info/definitions/1/0/access/objState">
+      <rdfs:label xml:lang="en">status</rdfs:label>
+      <rdfs:comment xml:lang="en">Describes the state of a resource, such as active or deleted.</rdfs:comment>
+      <rdfs:domain rdf:resource="http://fedora.info/definitions/v4/repository#Resource"/>
+      <rdfs:range rdf:resource="http://fedora.info/definitions/1/0/access/ResourceStatus"/>
+    </owl:ObjectProperty>
+
+    <owl:Class rdf:about="http://fedora.info/definitions/1/0/access/ResourceStatus">
+      <rdfs:label xml:lang="en">resource state</rdfs:label>
+      <rdfs:comment xml:lang="en">Values of the state property. The default values are active and deleted -- but additional values can be created.</rdfs:comment>
```

Comment:
Ok, I see where you are going.
I like "out-of-the-box" because it defines that these are just what are there now and you can build on as you like, without the inference that there needs to community support for changes to your use of this property. 
But as you have more experience with this, I will give you the final word: "community-supported" or "out-of-the-box"?